source/common/network/utility.cc

```diff
@@ -371,5 +371,17 @@ absl::uint128 Utility::flipOrder(const absl::uint128& input) {
   return result;
 }
 
+void Utility::addressToProtobufAddress(const Address::Instance& address,
```

Comment:
Already implemented here: https://github.com/envoyproxy/envoy/blob/master/source/common/access_log/grpc_access_log_impl.cc#L67 please consolidate

Also, a dedicated function like this needs dedicated tests.

Reply:
added tests and consolidated.